sdk/eventhubs/azure-messaging-eventhubs-stress/Dockerfile

```diff
@@ -23,17 +23,12 @@ RUN --mount=type=cache,target=/root/.m2 \
  mvn -f /stress-eh/sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/pom.xml clean install -Dcheckstyle.skip -Dgpg.skip -Dmaven.javadoc.skip -Drevapi.skip -Dspotbugs.skip -Djacoco.skip -DskipTests && \
  mvn -f /stress-eh/sdk/eventhubs/azure-messaging-eventhubs-stress/pom.xml clean install -Dcheckstyle.skip -Dgpg.skip -Dmaven.javadoc.skip -Drevapi.skip -Dspotbugs.skip -Djacoco.skip -DskipTests
 
-FROM mcr.microsoft.com/openjdk/jdk:11-mariner
+FROM mcr.microsoft.com/openjdk/jdk:21-mariner
 
 RUN yum update -y
 
 WORKDIR /app
 COPY --from=builder /stress-eh/sdk/eventhubs/azure-messaging-eventhubs-stress/target .
 
-ARG AGENT_URL=https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.4.13/applicationinsights-agent-3.4.13.jar
-ADD ${AGENT_URL} ./BOOT-INF/classes/
-
-ENTRYPOINT ["java", "-javaagent:BOOT-INF/classes/applicationinsights-agent-3.4.13.jar", \
-            "-jar", "/app/azure-messaging-eventhubs-stress/target/azure-messaging-eventhubs-stress-1.0.0-beta.1.jar", \
-            "--TEST_CLASS=EventSender"]
-
```

Comment:
I removed appinsights agent because it collected too much data and I had too little control over it. As a result it
- affected perf numbers
- significantly increased costs of stress tests


we now enable appinsights in code (TelemetryHelper) and via different means where we can enable things one-by-one.